core/src/locale/es.js

```diff
@@ -0,0 +1,69 @@
+export default {
+  '*': {
+    prefix: 'todos los',
+    suffix: '',
+    text: 'Desconocido',
+    '*': {
+      empty: { text: 'todos los {{ field.id }}' },
+      value: { text: '{{ val.text }}' },
+      range: { text: '{{ start.text }}-{{ end.text }}' },
+      everyX: { text: 'todos/as {{ every.value }}' }
+    },
+    meses: {
```

Comment:
Hi, sorry for the mistakes, still learning here... I commited the corrections but I'm not sure if I have to make a new pull request or this can work.

Reply:
No problem :)
Looks almost perfect, but I noticed one more detail.

There are still some English words for the cron expression `* * * * *`
![image](https://user-images.githubusercontent.com/8122139/213770794-087c4fe2-299e-496c-bc37-ade5a3c8a8a2.png)
This should be fixable by setting the following properties inside the localization object:
- `*.month.empty.text`
- `*.day.empty.text`
- `*.hour.empty.text`
- `*.minute.empty.text`

> not sure if I have to make a new pull request or this can work

No, you don't have to make a new PR. You could squash you commits ([How do I squash my last N commits together?](https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together)). If that is too much work, I can also squash them, when I merge this PR.